src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/BaseFunction.php

```diff
@@ -31,7 +31,22 @@ abstract class BaseFunction extends FunctionNode
      */
     protected array $nodes = [];
 
-    abstract protected function customizeFunction(): void;
+    /**
+     * TODO Make abstract when {@see customiseFunction()} is no more
```

Comment:
```suggestion
     * @internal
     * @see customiseFunction()
     * This method is meant for internal use only, and it is not suggested that the forks of the library depend on it.
     * It will be made abstract from version 3.0.
```

Reply:
Done